docs/userGuide/puml.md

```diff
@@ -15,11 +15,11 @@
 
 <box type="warning">
 
-**[Java](https://www.java.com/en/download/) must be installed to use this feature**
+**[Java](https://www.java.com/en/download/) must be installed to use this feature**. _(Any recent version should work)_
```

Comment:
How about "Java must be installed to generate the UML diagrams via a JAR executable"? 

To inform the user why is it needed.

Reply:
> * You have not initialised the `diagrams` folder within `_markbind` during `markbind init`. You also have to update the `Site.test.js` code since an additional folder is expected on initialisation. 

The diagram files can be in any folder, not necessarily restricted to the `diagrams` folder. I used `diagrams` in this case as an example, but it could have been any folder.